static/js/components/PodcastEpisodeCard_test.js

```diff
@@ -1,31 +1,34 @@
 // @flow
 /* global SETTINGS:false */
-import React from "react"
-import { shallow } from "enzyme"
 import { assert } from "chai"
 
 import { makePodcast, makePodcastEpisode } from "../factories/podcasts"
 import { embedlyThumbnail } from "../lib/url"
+import IntegrationTestHelper from "../util/integration_test_helper"
 
 import PodcastEpisodeCard, {
   PODCAST_IMG_WIDTH,
   PODCAST_IMG_HEIGHT
 } from "./PodcastEpisodeCard"
 
 describe("PodcastEpisodeCard", () => {
-  let podcast, episode
+  let podcast, episode, render, helper
   beforeEach(() => {
     podcast = makePodcast()
     episode = makePodcastEpisode(podcast)
+    helper = new IntegrationTestHelper()
+    render = helper.configureReduxQueryRenderer(PodcastEpisodeCard, {
+      podcast: podcast,
+      episode: episode
```

Comment:
formatting nit: you can declare this object like `{ podcast, episode }` because those variables are already bound to the name you want them to have in the object